Lib/pickletools.py

```diff
@@ -1540,7 +1549,9 @@ def __init__(self, name, code, arg,
       Stack before:  ... pylist markobject stackslice
       Stack after:   ... pylist+stackslice
 
-      although pylist is really extended in-place.
+      although pylist is really extended in-place. The .extend() attribute
+      function is attempted first, and if that fails the .append() attribute
+      is ran instead.
```

Comment:
It is not clear. It can be interpreted that the call of `extend()` is attempted first, and if it fails, `append()` is used.

```suggestion
      although pylist is really extended in-place. The extend() method is
      used if it exists, otherwise the append() method is used.
```